docs/deployment/README.md#cumulus-instance-sizing

```diff
@@ -422,9 +422,20 @@ Notes on specific variables:
 
 Consider [the sizing of your Cumulus instance](#cumulus-instance-sizing) when configuring your variables.
 
-#### Configure the Thin Egress App
+#### Choose a distribution API
 
-The Thin Egress App is used for Cumulus distribution. Follow the steps [in the documentation](./thin_egress_app) to configure distribution in your `cumulus-tf` deployment.
+Cumulus can be configured to use either the Thin Egress App (TEA) or the Cumulus Distribution API. The default selection is the Thin Egress App if you're using the [Deployment Template](https://github.com/nasa/cumulus-template-deploy).
+
+**IMPORTANT** If you already have a deployment using the TEA distribution and want to switch to Cumulus Distribution, there will be an API Gateway change. This means that there will be downtime while you update your CloudFront endpoint to use
```

Comment:
Is this NGAP specific?

Reply:
Oh interesting. Would it not be true? I was thinking that you'd be creating a new API gateway regardless and doing so would mean _some_ period of time switching from the old TEA gateway to the new one. Am I thinking about this wrong?